Altis_Life.Altis/core/configuration.sqf

```diff
@@ -47,8 +47,6 @@ life_firstSpawn = true;
 //Settings
 life_settings_enableNewsBroadcast = profileNamespace getVariable ["life_enableNewsBroadcast", true];
 life_settings_enableSidechannel = profileNamespace getVariable ["life_enableSidechannel", true];
-life_settings_tagson = profileNamespace getVariable ["life_settings_tagson", true];
-life_settings_revealObjects = profileNamespace getVariable ["life_settings_revealObjects", true];
```

Comment:
This seemed to bring more clarity than inlining?

Reply:
The variables were only used twice, which is pointless to store for when its used so rarely.